proposals/0001-simd-process.md

```diff
@@ -0,0 +1,170 @@
+
+  - **Core**: Anything that affects consensus or substantial changes to the
+validator.
+  - **Networking**: Changes or substantial improvements to network protocol
+specifications.
+  - **Interfaces**: Improvements around the client JSON RPC API specifications
+and standards. 
+
+- A **Meta** Proposal describes a process surrounding Solana or proposes a
+change to (or an event in) a process. Process Proposals are like Standard
+Proposals but apply to areas other than the Solana protocol itself. They may
+propose an implementation, but not to Solana's codebase; they often require
+community consensus; unlike Informational Proposals, they are more than
+recommendations, and users are typically not free to ignore them. Examples
+include procedures, guidelines, changes to the decision-making process, and
+changes to the tools or environment used in Solana development. Any meta-EIP is
+also considered a Process Proposals.
+
+## Proposal Lifecycle
+
+The stages in a lifecycle of a proposal are as follows:
+
+- Idea
+- Draft
+- Review
+- Accepted
+- Stagnant
+- Withdrawn
+- Implemented
+
+```mermaid
+flowchart LR
+  Idea
+  Draft
+  Review
+  Accepted
+
+  subgraph fail[&nbsp];
+    Stagnant
+    Withdrawn
+  end
+
+  style fail fill:#ffe8e7,stroke:none
+  style Accepted fill:#f3f8dc,stroke:#c3db50;
+
+  Idea ---> Draft;
+  Draft ---> Review;
+  Review ---> Accepted;
+
+  Draft ---> Stagnant;
+  Review ---> Stagnant;
+  Review ---> Withdrawn;
+```
+
+### Idea
+
+At the idea stage, parties involved in the proposal are you, the champion or
+proposal author, the reviewers, and the Solana Core Contributors.
+
+Before you begin writing a formal proposal, you should vet your idea. Ask the
+Solana core community first if an idea is original to avoid wasting time on
+something that will be rejected based on prior research. It is thus recommended
+to discuss the proposal on the Solana Tech Discord under the #core-technology
+channel.
+
+### Draft
+
+To begin drafting the proposal, do the following:
+
+- Fork the proposal repository
+- Copy `proposals/XXXX-template.md` to `proposals/XXXX-my-feature.md` (where
+"my-feature" is descriptive)
+- Fill in the proposal. Put care into the details: proposals that do not
+present convincing motivation, demonstrate lack of understanding of the
+design's impact, or are disingenuous about the drawbacks or alternatives tend
+to be poorly received.
+- Submit a pull request.
+- Now that your proposal has an open pull request, use the issue number of the
+PR to update the `XXXX-` prefix to the number.
+
+### Review
+
+During review, the owner of the proposal is in charge of gathering and
+integrating feedback. The most relevant core contributors to the proposal
+should be included in the review process. Review will take place completely
+through Github so that all comments are collected and documented. Once
+consensus is met by the core contributors, the proposal can either be accepted
```

Comment:
How is consensus reached or measured? This seems like the most important part of this new process.

Reply:
I have added more information about consensus. While it is important, I am trying to avoid defining what consensus looks like, other than as a social construct. I want the process of proposals to be very similar to how it is today, driven by the norms of core contributors, and not impose more structure than is necessary.

Reply:
I think consensus on proposals, at least on the activation thereof, should be migrated to on-chain voting by means of stake-weighted realms DAO. Right now the on-chain voting is severely lacking both in application and functionality, as it requires a cumbersome CLI-driven token transfer and provides no opportunity to dissent and little access or insight into voting record or history.

A Realms DAO can be created where voting power is synchronized to stake-weight every epoch, feature activation should require a majority of stake to vote in favour of it. Currently the momentum of upgrading to a feature-including version forces validators to comply with no opportunity to dissent, the only option is "don't adopt a change you don't agree with" which however means suicide as one can no longer participate in the chain. An opportunity to openly and transparently vote and express assent or dissent is needed.

Reply:
@michaelh-laine I agree with your sentiment on wanting to be able to have a voice and that there should be more around feature activations. For this proposal I'd like to stay on the process of proposals vs the implementation and activation.